src/crimson/tools/store_bench/store-bench.cc

```diff
@@ -0,0 +1,685 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
+
+/**
+ * crimson-store-bench
+ *
+ * This tool measures various IO patterns against the crimson FuturizedStore
+ * interface.
+ *
+ * Usage should be:
+ *
+ * crimson-store-bench --store-path <path>
+ *
+ * where <path> is a directory containing a file block.  block should either
+ * be a symlink to an actual block device or a file truncated to an appropriate
+ * size if performance isn't relevant (testing or developement of this utility,
+ * for instance).
+ *
+ * One might want to add something like the following to one's .bashrc to
+ * quickly run this utility during development from build/:
```

Comment:
Either this paragraph should be removed or the bashrc function should be readded.  I'd suggest the latter.